src/api/app/policies/project_policy.rb

```diff
@@ -35,6 +35,10 @@ def unlock?
     user.can_modify?(record, true)
   end
 
+  def source_access?
+    !User.current.is_admin? && record.disabled_for?('sourceaccess', nil, nil) ? false : true
```

Comment:
isn't this a rather fancy way to implement the not operator ? :)

Reply:
ops, fixed, would you mind to check it again @coolo ?